src/test/java/org/kohsuke/github/GHRepositoryTest.java

```diff
@@ -47,6 +47,18 @@ private GHRepository getRepository(GitHub gitHub) throws IOException {
         return gitHub.getOrganization("hub4j-test-org").getRepository("github-api");
     }
 
+    /**
+     * Test sync of fork
+     *
+     * @throws IOException
+     *             Signals that an I/O exception has occurred.
+     */
+    @Test
+    public void sync() throws IOException {
+        GHRepository r = getRepository();
+        r.sync("main");
```

Comment:
Add slightly more complex tests to cover the returned record and also the error/exception case.  (You can do all in one method if you want or separately.)